### ISSUE TITLE
Docker: cp: cannot stat '/genesis.json': No such file or directory

### ISSUE THREAD

**Issue by kenorb** [EOSIO/eos#2801]
```
$ docker run -it eosio/eos nodeosd.sh
cp: cannot stat '/genesis.json': No such file or directory
3300451ms thread-0   chain_plugin.cpp:103          plugin_initialize    ] initializing chain plugin
3300452ms thread-0   http_plugin.cpp:247           plugin_initialize    ] configured http to listen on 0.0.0.0:8888
3300452ms thread-0   wallet_plugin.cpp:41          plugin_initialize    ] initializing wallet plugin
...
```

```
# nodeosd.sh --help
cp: cannot stat '/genesis.json': No such file or directory
```

**Comment by ray0807** [comment]
Docker image eosio/eos  can not work normally !
![image](https://user-images.githubusercontent.com/7233544/39688552-deae0a4a-5205-11e8-958b-5d0b39ffef44.png)
But my mac can't install eos , so I build a image in docker ( I used  eos/builder image).
https://github.com/xmtj/eos_dev

**Comment by ericiles** [comment]
This has been resolved with PR #2825.
